src/Server/AllServers.ts

```diff
@@ -188,13 +190,14 @@ export function initForeignServers(homeComputer: Server): void {
 
 export function prestigeAllServers(): void {
   for (const member of Object.keys(AllServers)) {
+    // eslint-disable-next-line @typescript-eslint/no-dynamic-delete
     delete AllServers[member];
   }
   AllServers = {};
 }
 
 export function loadAllServers(saveString: string): void {
-  AllServers = JSON.parse(saveString, Reviver);
+  AllServers = JSON.parse(saveString, Reviver) as typeof AllServers;
```

Comment:
I think I've actually seen a crash on load due to this specific type assumption. You don't have to fix it now, but I'd rather not paper over it with a cast.